### PR TITLE
F t31115 remove translator from dp select

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T31115

- add Trans-Key to translations.js
- remove Translator.trans from DpSelect.

### CODE REVIEW

File: src/components/DpSelect/DpSelect.vue [hunk 1]
```diff
@@ -99,7 +100,7 @@ export default {
     placeholder: {
```
Comment (spiess-demos) [inline src/components/DpSelect/DpSelect.vue:100]: We do not need the whole prop anymore.

File: src/components/DpSelect/DpSelect.vue [hunk 2]
```diff
@@ -21,21 +21,22 @@
         disabled
         value=""
         :selected="selected === ''">
-        {{ Translator.trans(placeholder) }}
+        {{ selectPlaceholder }}
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply src/components/DpSelect/DpSelect.vue:24]: Unfortunately it doesn't work that way
When executing the code in this way shows us a errer
```
Missing import statement 
[Vue warn]: Error in render: "TypeError: e.de is undefined"

found in

---> <DpSelect>
       <ChangeStateAtDate> at client/js/components/news/ChangeStateAtDate.vue
         <Root>
```
that's why I made it as computed!

https://github.com/demos-europe/demosplan-ui/pull/357/commits/343248cd224d7253cf310e1296cd422e79dd4d5a

File: src/components/DpSelect/DpSelect.vue (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpSelect/DpSelect.vue]: Done in https://github.com/demos-europe/demosplan-ui/pull/357/commits/44e95c1ccaeef70ce8bd71820e718d91bd6dea8c

**Review by spiess-demos — CHANGES_REQUESTED** [review]
A CHANGELOG entry is missing, apart from that, lgtm.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Removed
+
+- ([#357](https://github.com/demos-europe/demosplan-ui/pull/357)) Remove Translator from DpSelect ([@ahmad-demos](https://github.com/ahmad-demos))
+
 - ([#344](https://github.com/demos-europe/demosplan-ui/pull/344)) Add documentation for DpTabs ([@ahmad-demos](https://github.com/@ahmad-demos))
 - ([#333](https://github.com/demos-europe/demosplan-ui/pull/333)) Add "rounded" prop to DpButton, add "arrow-up" and "arrow-down" icons to DpIcon ([@spiess-demos](https://github.com/spiess-demos))
```
Comment (spiess-demos) [inline CHANGELOG.md:12]:
```suggestion
### Removed

- ([#357](https://github.com/demos-europe/demosplan-ui/pull/357)) Remove Translator from DpSelect ([@ahmad-demos](https://github.com/ahmad-demos))

### Added

- ([#344](https://github.com/demos-europe/demosplan-ui/pull/344)) Add documentation for DpTabs ([@ahmad-demos](https://github.com/@ahmad-demos))
- ([#333](https://github.com/demos-europe/demosplan-ui/pull/333)) Add "rounded" prop to DpButton, add "arrow-up" and "arrow-down" icons to DpIcon ([@spiess-demos](https://github.com/spiess-demos))
```

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**